tonic/src/transport/server/conn.rs

```diff
@@ -98,6 +98,36 @@ impl Connected for TcpStream {
     }
 }
 
+/// Connection info for Unix domain socket streams.
+///
+/// This type will be accessible through [request extensions][ext] if you're using
+/// a unix stream.
+///
+/// See [`Connected`] for more details.
+///
+/// [ext]: crate::Request::extensions
+#[cfg(unix)]
```

Comment:
Maybe we can just move this to a `unix.rs` file and `cfg` that whole file?

Reply:
Sounds good to me 👍🏽